solarwinds_apm/apm_config.py

```diff
@@ -485,9 +483,71 @@ def get(self, key: str, default: Any = None):
         )
         return value if value is not None else default
 
-    def update_with_cnf_file(self, cnf_path: str) -> None:
-        """Update the settings with the config file, if any."""
-        # TODO Implement config with cnf_file after alpha
+    def update_with_cnf_file(self) -> None:
+        """Update the settings with the config file (json), if any."""
+        cnf_filepath = os.environ.get(
+            "SW_APM_CONFIG_FILE", "./solarwinds-apm-config.json"
+        )
+        if not cnf_filepath:
+            return
+        cnf_dict = None
+        try:
+            with open(cnf_filepath, encoding="utf-8") as cnf_file:
+                try:
+                    cnf_dict = json.load(cnf_file)
+                except ValueError as ex:
+                    logger.error(
+                        "Invalid config file, must be valid json. Ignoring: %s",
+                        ex,
+                    )
+                    return
+        except FileNotFoundError as ex:
+            logger.error("Invalid config file path. Ignoring: %s", ex)
+            return
+
+        try:
+            val = cnf_dict.get("transaction").get("prependDomain")
+            if val is not None:
+                self._set_config_value("transaction.prepend_domain_name", val)
+        except AttributeError:
+            pass
+        available_cnf = set(self.__config.keys())
+        # TODO after alpha: is_lambda
+        for key in available_cnf:
+            cnf_key_parts = key.split("_")
+            cnf = f"{cnf_key_parts[0]}{''.join(part.title() for part in cnf_key_parts[1:])}"
```

Comment:
Just to make sure I understand this goes from `snake_case` to `camelCase` ?

Reply:
Yes 👍

Reply:
Maybe worth extracting to a function in a future revision to make it more evident at first glance